src/plopp/backends/matplotlib/canvas.py

```diff
@@ -25,6 +25,25 @@ def _none_if_not_finite(x: Union[float, int, None]) -> Union[float, int, None]:
     return x if np.isfinite(x) else None
 
 
+def _cursor_value_to_variable(
+    x: Union[float, int], dtype: sc.DType, unit: str
+) -> sc.Variable:
+    if dtype == sc.DType.datetime64:
+        # Annoying chain of conversion but matplotlib has its own way of converting
+        # dates to numbers (number of days since epoch), and num2date returns a python
+        # datetime object, while scipp expects a numpy datetime64.
+        return sc.scalar(np.datetime64(mdates.num2date(x).replace(tzinfo=None))).to(
```

Comment:
We typically say that scipp datetimes are in UTC. But this looks like the result will be in whatever timezone matplotlib uses but without timezone info. I am assuming mpl uses the local timezone unless specified otherwise in the input. Is that correct?

Did you check that the cursor label is consistent with the axis labels regardless of timezone? And the same for zooming, selecting rectangles, etc.

Reply:
I think that matplotlib defaults to UTC:

> All the Matplotlib date converters, locators and formatters are timezone aware. If no explicit timezone is provided, `rcParams["timezone"]` (default: 'UTC') is assumed,

in https://matplotlib.org/stable/api/dates_api.html#matplotlib-date-format.

In the examples I used, the first point read from the cursor corresponds to the first value in the array:
![Screenshot at 2024-03-08 13-39-25](https://github.com/scipp/plopp/assets/39047984/d39897a9-1f69-4a5d-97c3-34bb0010aefa)

Reply:
Ok. And zooming and selecting points with mpl toolbox all work?

Reply:
Seems like it, yes
![Figure 1 (3)](https://github.com/scipp/plopp/assets/39047984/e3f181d4-44ee-405a-b512-b95d528ca693)